crates/admin/Cargo.toml

```diff
@@ -8,7 +8,7 @@ license.workspace = true
 publish = false
 
 [features]
-default = ["replicated-loglet", "serve-web-ui"]
+default = ["replicated-loglet", "serve-web-ui", "memory-loglet"]
```

Comment:
Why making the `memory-loglet` feature a default feature?

Reply:
I had an issue with memory-loglet and features and this was a temp solution. I will take it out.